spring-cloud-stream-binder-rocketmq/src/main/java/com/alibaba/cloud/stream/binder/rocketmq/integration/RocketMQMessageHandler.java

```diff
@@ -97,6 +111,24 @@ public void start() {
 						.build(), e);
 			}
 		}
+		if (producerProperties.isPartitioned()) {
+			try {
+				List<MessageQueue> messageQueues = rocketMQTemplate.getProducer()
+						.fetchPublishMessageQueues(destination);
+				if (producerProperties.getPartitionCount() != messageQueues.size()) {
+					logger.info(String.format(
+							"The partition count will change from '%s' to '%s'",
```

Comment:
log need add topic info, maybe "The `partitionCount` of topic '%s' will change from ..." is better.